tests/integrations/tests/test_bedrock.py

```
@@ -51,24 +66,133 @@
+        "bedrock",
+        region_name=region,
+        endpoint_url=base_url,
+    )
+
+    # Add provider-specific header to all requests
+    client.meta.events.register('before-send', create_provider_header_handler(provider))
+
+    return client
+
+
+def create_bedrock_batch_jsonl(model_id: str, num_requests: int = 2) -> str:
```

Ruff (0.14.7)

Unused function argument: `model_id`

(ARG001)